src/pages/workspace/categories/CategoryGLCodePage.tsx

```diff
@@ -0,0 +1,95 @@
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type SCREENS from '@src/SCREENS';
+import INPUT_IDS from '@src/types/form/WorkspaceCategoryForm';
+import type {PolicyCategories} from '@src/types/onyx';
+
+type WorkspaceEditCategoryGLCodePageOnyxProps = {
+    /** Collection of categories attached to a policy */
+    policyCategories: OnyxEntry<PolicyCategories>;
+};
+
+type EditCategoryPageProps = WorkspaceEditCategoryGLCodePageOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.CATEGORY_GL_CODE>;
+
+function CategoryGLCodePage({route, policyCategories}: EditCategoryPageProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+    const categoryName = route.params.categoryName;
+    const glCode = policyCategories?.[categoryName]?.glCode;
+    const {inputCallbackRef} = useAutoFocusInput();
+
+    const editGLCode = useCallback(
+        (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_CATEGORY_FORM>) => {
+            const newGLCode = values.glCode.trim();
+            if (newGLCode !== glCode) {
+                Category.updatePolicyCategoryGLCode(route.params.policyID, categoryName, newGLCode);
+            }
+            Navigation.goBack();
+        },
+        [categoryName, glCode, route.params.policyID],
+    );
+
+    return (
+        <AccessOrNotFoundWrapper
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            policyID={route.params.policyID}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CATEGORIES_ENABLED}
+        >
+            <ScreenWrapper
+                includeSafeAreaPaddingBottom={false}
+                style={[styles.defaultModalContainer]}
+                testID={CategoryGLCodePage.displayName}
+                shouldEnableMaxHeight
+            >
+                <HeaderWithBackButton
+                    title={translate('workspace.categories.glCode')}
+                    onBackButtonPress={() => Navigation.goBack(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(route.params.policyID, route.params.categoryName))}
+                />
+                <FormProvider
+                    formID={ONYXKEYS.FORMS.WORKSPACE_CATEGORY_FORM}
+                    onSubmit={editGLCode}
+                    submitButtonText={translate('common.save')}
+                    style={[styles.mh5, styles.flex1]}
+                    enabledWhenOffline
+                >
+                    <InputWrapper
+                        ref={inputCallbackRef}
+                        InputComponent={TextInput}
+                        defaultValue={glCode}
+                        label={translate('workspace.categories.glCode')}
+                        accessibilityLabel={translate('workspace.categories.glCode')}
+                        inputID={INPUT_IDS.GL_CODE}
+                        role={CONST.ROLE.PRESENTATION}
+                        maxLength={CONST.MAX_LENGTH_256}
```

Comment:
Hmm why not `CONST.FORM_CHARACTER_LIMIT`? I didn't look at the doc recently, but i imagine GL codes are never > 50 characters in length?

Reply:
@Beamanator if im not wrong, in database the limit is 256 chars

Reply:
Merp sorry for not responding here - you def may be right, but the better question to me is: Can GL codes ever get anywhere near 50 characters even? If not, I'd prefer we limit to 50 chars

Reply:
@Gonals any opinion here? 🤷